extensions/panache/hibernate-orm-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/orm/rest/data/panache/deployment/CustomPathItemsResource.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.hibernate.orm.rest.data.panache.deployment;
+
+import java.util.List;
+
+import javax.ws.rs.core.Response;
+
+import io.quarkus.hibernate.orm.rest.data.panache.PanacheEntityCrudResource;
+import io.quarkus.rest.data.panache.OperationProperties;
+import io.quarkus.rest.data.panache.ResourceProperties;
+
+@ResourceProperties(path = "custom-items", hal = true)
+public interface CustomPathItemsResource extends PanacheEntityCrudResource<Item, Long> {
```

Comment:
So you were against CRUD in the module name, but not here in the resource name? Should it be `PanacheEntityDataResource`?

Reply:
It does sounds better to me without CRUD. I'll try to rename it quickly.

Reply:
Maybe just `PanacheEntityResource`?

Reply:
I guess, yes.

Reply:
Probably the same simplification could be done to the repository base name?

Reply:
Yes of course. Will do `PanacheEntityResource` and `PanacheRepositoryResource` for the ones defined in the Hibernate extension. And `RestDataResource` for that one defined in the common module that is extended by the first two.

Reply:
:+1:

Reply:
Done